tutorials/plot_source_alignment.py

```diff
@@ -135,3 +203,5 @@
 # to warp a subject (usually ``fsaverage``) to subject digitization data, see
 # `these slides
 # <http://www.slideshare.net/mne-python/mnepython-scale-mri>`_.
+#
+# .. _trans_matrices: https://en.wikipedia.org/wiki/False_discovery_rate
```

Comment:
does not seem to be the correct link, this one belongs to the beautiful Statistical Inference docs :)

Reply:
(Agree, the stats docs are great.)